src/lib/times.js

```diff
@@ -77,7 +78,8 @@ export function confirmationTime(time, timeType) {
   if (upcoming.minutes()) {
     timeStr = getLocalizedDateTime(upcoming, 'confirmation_time');
   }
-  const dateStr = getLocalizedDateTime(upcoming, 'confirmation_date');
+  const dateStr = getLocalizedDateTime(upcoming,
+    (upcoming.year() === thisYear) ? 'confirmation_date' : 'confirmation_date_with_year');
```

Comment:
Could we switch this to the `let`/`if()` like timeStr?  I'm kinda allergic to the ternary `?:` operator…  (I mean, not totally allergic, but it often seems a little too clever for its own good.  :wink:)